tests/test_sklearn_explain_weights.py

```diff
@@ -270,15 +270,18 @@ def test_explain_linear_feature_re(newsgroups_train, vec):
         vec = InvertableHashingVectorizer(vec)
         vec.fit(docs)
 
-    res = explain_weights(clf, vec=vec, feature_re='^ath')
+    res = explain_weights(
+        clf, vec=vec, feature_re='^ath')
```

Comment:
Hah, that line suffered through some changes :) Thank for the catch! Fixed in 724c63a